low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -157,25 +150,11 @@ class SSPLBundleGenerate(object):
             with open(self.ipmi_sel_data, 'w+') as sel_list:
                 sel_list.write(response)
 
-    def get_consul_data(self):
-        self.consul_folder_name = f"consul_sspl_data-{str(int(time.time()))}"
-        consuldump = f"{CONSUL_DUMP} -n {self.consul_folder_name} -l {sspl_bundle_tmp}/ -d"
-        consuldump_cmd = {
-            "sspl_config": f"{consuldump} consul_sspl_config sspl/",
-            "sspl_cache": f"{consuldump} consul_sspl_cache var/{PRODUCT_FAMILY}/sspl",
-            "system_information": f"{consuldump} consul_system_information system_information/",
-            "storage_enclosure": f"{consuldump} consul_storage_enclosure storage_enclosure/"
-        }
-        for key, value in consuldump_cmd.items():
-            subout = self._run_command(value)
-            if subout[2] != 0:
-                print(f"Unable to get consul data of {key}...!")
-            del subout
-        consul_folder_path = f"{sspl_bundle_tmp}/{self.consul_folder_name}"
-        self.consul_sspl_config = f"{consul_folder_path}/consul_sspl_config"
-        self.consul_sspl_cache = f"{consul_folder_path}/consul_sspl_cache"
-        self.consul_system_information = f"{consul_folder_path}/consul_system_information"
-        self.consul_storage_enclosure = f"{consul_folder_path}/consul_storage_enclosure"
+    def get_config_data(self):
+        if not os.path.exists(self.sspl_conf_dir):
+            os.makedirs(self.sspl_conf_dir)
```

Comment:
This can be simplified into one line as said above.

Reply:
Done